### PR TITLE
Release v4.0.43.1

### PR DESCRIPTION
**Pull request by hy-ca**

### CODE REVIEW

File: site/pages/docs/versions/dwnld-en.hbs [hunk 1]
```diff
@@ -25,12 +25,12 @@
 		<tbody>
 			<tr>
 				<td>Web Experience Toolkit (WET)</td>
-				<td>v4.0.43 (STR)</td>
+				<td>v4.0.43.1 </td>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-en.hbs:28]: Can you keep (STR)

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 2]
```diff
@@ -25,12 +25,12 @@
 		<tbody>
 			<tr>
 				<td>Boîte à outils de l'expérience Web (WET-BOEW)</td>
-				<td>v4.0.43 (DCT)</td>
+				<td>v4.0.43.1 </td>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-fr.hbs:28]: Can you keep (DCT)

**Review by duboisp — CHANGES_REQUESTED** [review]

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 3]
```diff
@@ -4,7 +4,7 @@
 	"language": "fr",
 	"description": "Téléchargements de la dernière version - Boîte à outils de l'expérience Web (BOEW)",
 	"altLangPrefix": "dwnld",
-	"dateModified": "2021-06-03"
+	"dateModified": "2021-06-016"
```
Comment (RobJohnston) [inline site/pages/docs/versions/dwnld-fr.hbs:7]: 2021-06-016 -> 2021-06-16

**Comment by hy-ca** [comment]
Hi Pierre,

I have amended the last commit; however, the files are not updated in pull request yet. I have repeated the process, while am wondering whether I should wait for pull request to be updated. Let me know if further actions are required.

***@***.***D76359.6B51C880]
***@***.***D76359.6B51C880]

***@***.***D76359.6B51C880]


Sincerely,
Helen Q. Yu

IM/IT Analyst Programmer, Digistal Service Directorate
Citizen Service Branch / Service Canada
***@***.******@***.***>  / Tel: 647-228-5794
Programmeur analyste GI TI, Direction du service numérique
Direction générale de service aux citoyens / Service Canada
***@***.******@***.***>  / Tél: 647-228-5794

**Comment by hy-ca** [comment]
Hi Pierre,

Sorry to bother you again.

I have tried git push -f to the release branch and received the following messages. I was able to push it to origin master and hoping it can be worked out some way.

***@***.***D7635C.AB101D20]


***@***.***D7635C.AB101D20]

Sincerely,
Helen Q. Yu

IM/IT Analyst Programmer, Digistal Service Directorate
Citizen Service Branch / Service Canada
***@***.******@***.***>  / Tel: 647-228-5794
Programmeur analyste GI TI, Direction du service numérique
Direction générale de service aux citoyens / Service Canada
***@***.******@***.***>  / Tél: 647-228-5794



From: Yu, Helen Q [NC]
Sent: June 17, 2021 9:16 AM
To: 'wet-boew/wet-boew' ***@***.***>
Subject: RE: [wet-boew/wet-boew] Release v4.0.43.1 (#9126)

Hi Pierre,

I have amended the last commit; however, the files are not updated in pull request yet. I have repeated the process, while am wondering whether I should wait for pull request to be updated. Let me know if further actions are required.

***@***.***D7635C.AB101D20]
***@***.***D7635C.AB101D20]

***@***.***D7635C.AB101D20]


Sincerely,
Helen Q. Yu

IM/IT Analyst Programmer, Digistal Service Directorate
Citizen Service Branch / Service Canada
***@***.******@***.***>  / Tel: 647-228-5794
Programmeur analyste GI TI, Direction du service numérique
Direction générale de service aux citoyens / Service Canada
***@***.******@***.***>  / Tél: 647-228-5794

**Review by duboisp — APPROVED** [review]

File: site/pages/docs/versions/dwnld-fr.hbs (hunk 3 above)
Comment (hy-ca) [reply site/pages/docs/versions/dwnld-fr.hbs]: Thank you and it has been resolved.

**Pull request merged**